public/app/components/codeEditor/codeEditor.js

```diff
@@ -34,6 +34,17 @@ angular.module('kibibitCodeEditor')
       vm.undoManager = _editor.getSession().getUndoManager();
       SettingsService.settings.currentUndoManager = vm.undoManager;
       SettingsService.settings.currentEditor = _editor;
+      var editorSettings = SettingsService.settings.editorSettings;
+      _editor.setOptions({
```

Comment:
kbCodeEditor should have a `init` function that will get a settings object and will init all the variables. This shouldn't be inline inside ace loaded because we might use this function not on load, or we would want to use it not with the global settings